### ISSUE TITLE
[Feature] Support sending files to Claude Sonnet-3.5

### ISSUE THREAD

**Issue by mnhcorp** [Bin-Huang/chatbox#1450]
**Problem Description**
Sending files to Claude-3.5 is not supported. Only Chatbox AI 4 is supported.

**Proposed Solution**
Allow sending PDF/text/csv/etc files to Claude 3.5 sonnet. Perhaps the app could cover these files to text and append to the prompt? I don't think Anthropic's API supports uploading files (I could be wrong).

**Additional Context**
None.

**Comment by codeconuts** [comment]
I also agree that this feature should be added. Also yes, Anthropic API doesn't support uploading files for now.

**Comment by Fe2-O3** [comment]
https://github.com/anthropics/anthropic-cookbook/blob/main/misc/pdf_upload_summarization.ipynb

it's doable apparently.

**Comment by phammer** [comment]
Yes, would be great to have that support for all file types as listed on the website: pdf, doc, ppt, xls.
